Dockerfile

```diff
@@ -14,23 +14,20 @@
 
 FROM ubuntu:18.04
 
+MAINTAINER soda team
+
 RUN apt-get update -y && \
     apt-get install -y python3-pip && \
     apt-get install -y sqlite3 && \
-    apt-get install -y libffi-dev
+    apt-get install -y libffi-dev && \
+    pip3 install --upgrade pip
```

Comment:
Can we use a custom base image with all these dependencies pre-installed in it to reduce the number of layers and build time. (Just a keep a seprate docker file to make this base image for reference to the developers to see what goes in the base image)
Ref : https://docs.docker.com/develop/dev-best-practices/

Reply:
Yes, The suggestions are valid but the PR/Issues is opened for Fixing Dockerfile+Docker-compose issues

Reply:
Opened an issue #553